test/functionalTest/cases/2015_notification_templates/notification_templates_with_invalid_chars.test.DISABLED

```diff
@@ -1,40 +0,0 @@
-# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand this .test has been covered by other .test in previous PRs. Is that correct?